spec.html

```diff
@@ -35492,8 +35492,8 @@ <h1>get %TypedArray% [ @@species ]</h1>
     </emu-clause>
 
     <emu-clause id="sec-properties-of-the-%typedarrayprototype%-object">
-      <h1>Properties of the %TypedArray.prototype% Object</h1>
-      <p>The <dfn>%TypedArray.prototype%</dfn> object:</p>
+      <h1>Properties of the %TypedArray% Prototype Object</h1>
+      <p>The <dfn>%TypedArray% prototype object</dfn>:</p>
       <ul>
         <li>has a [[Prototype]] internal slot whose value is %Object.prototype%.</li>
         <li>is an ordinary object.</li>
```

Comment:
This should keep the `<dfn>` for `%TypedArray.prototype%`:
```suggestion
        <li>is <dfn>%TypedArray.prototype%</dfn>.</li>
        <li>is an ordinary object.</li>
        <li>has a [[Prototype]] internal slot whose value is %Object.prototype%.</li>
```